client/App.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import $ from 'jquery';
+
+import Stats from './components/Stats.jsx';
+import Sort from './components/Sort.jsx';
+import ReviewsFeed from './components/ReviewsFeed.jsx';
+
+class App extends React.Component {
```

Comment:
Have you looked into Hooks? They're just as functional as class components and make states a lot easier/faster to build and pass between sub-components. Not required by any means, but I'd encourage you to look into it! They're really quick to pick up.

Reply:
@CSellers720 - Thank you for this comment! I don't have any experience with React Hooks yet, so figured I'd start without them. August's presentation inspired me to learn Hooks though, so was planning on refactoring later this week if there was time. I'll give them a glance today before I go any further and make a call of whether I want to use them or not.